store.go

```diff
@@ -257,15 +257,15 @@ func (s *FilesystemStore) save(session *Session) error {
 	if err != nil {
 		return err
 	}
-	filename := filepath.Join(s.path, "session_"+session.ID)
+	filename := filepath.Join(s.path, filepath.Base("session_"+session.ID))
```

Comment:
Why not `"session_" + filepath.Base(session.ID)`?

Reply:
Strictly speaking I think this would still work, however modifying already canonicalized values can often lead to bypasses, so it feels more correct to `filepath.Join` the return value of `filepath.Base` to ensure we get a single path segment.

Reply:
But if one manages to smuggle a / into `session.ID` it'll return a base without `session_`.

https://go.dev/play/p/syqwxJC5tzF

Reply:
That's is a good point, so perhaps it would be better to do it this way.

Reply:
Imagine a hypothetical system where `_` is a path separator. I fear that `filepath.Join` is inherently unsafe